site/src/components/BlogPosts.scss

```diff
@@ -0,0 +1,55 @@
+
+@import "../scss/init";
+
+.BlogPosts{
+  margin: rem-calc(24) 7.5%;
+  h2 {
+    padding-bottom: rem-calc(26);
+    background-image: url(../../src/img/down-chevrons.svg);
+    background-position: 50% 100%;
+    background-repeat: no-repeat;
+    text-align: center;
+    font-family: $body-font-family;
+    font-size: rem-calc(18);
+    line-height: 1.222222222;
+    font-weight: bold;
+  }
+  ul {
+    @include breakpoint(640px) {
+      display: flex;
+      justify-content: space-between;
+    }
+    margin: rem-calc(82) auto;
+    max-width: rem-calc(1280);
+    list-style: none;
+    li {
+      @include breakpoint(640px) {
+        width: 31.5%;
+      }
+      text-align: center;
+      a {
+        img {
+          border: 1px dashed $blue-gray;
+          margin-bottom: rem-calc(45);
+        }
+        span {
+          display: block;
+          font-weight: bold;
+          &.BlogPosts__tag {
+            margin-bottom: rem-calc(10);
+            color: $minty;
+            text-transform: uppercase;
+            font-size: rem-calc(16);
+            line-height: 1.1875;
```

Comment:
Is there a reason not to use rem-calc to derive the line height here?  Line height as a ratio of font size is hard (for me) to grok and makes it tougher to see how it needs to be adjusted in response to future changes.

Reply:
I always prefer doing it as a ratio. And I've been doing this across the LCM site too… 😬 . I'm open to switching to rem-calc derived pixel-based line-heights, but this is good place to explain my thinking.

1. If there is a call to change the font-size without changing how the spacing of the type feels then I can do it just by changing the font-size and don't even need to look at the line-height. But if the line-height is set in a fixed unit size like rem-calc derived pixels and I want to change the font-size I need to (1) change the font-size, (2) figure out the ratio of the original font-size to the original line-height, and then (3) change the new line-height to according to that ratio relative to the new font-size.

2. If you can learn to read read line-heights as a ratio you can know the _feel_ of text spacing without needing to consider the font-size. For example: 1 = no space between lines at all, 1.15-1.4 = pretty tight, 1.5 to 2 = normal body copy, >2 = huge space between lines.

I've been waiting for a good context to explain my thinking on this, because I don't think I'm doing it in the most common way, but I really like this approach for the two reasons listed above.

Reply:
Ok. I can get behind it.  It doesn't intuitively make sense to me, but I see your points, and either way seems about the same level of difficulty to adjust.